Extensions/FileSystem/filesystemtools.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @memberof gdjs
+ * @class filesystem
+ * @static
+ * @private
+ */
+
+gdjs.filesystem = {}
+
+/**
+ * Get the path to 'Desktop' folder.
+ * @return {string} The path to the desktop folder
+ */
+gdjs.filesystem.getFolderDesktop = function (runtimeScene) {
+  const electron = runtimeScene.getGame().getRenderer().getElectron();
+
+  if (electron) {
+    return electron.getPath('desktop') ? electron.getPath('desktop') : '';
```

Comment:
Could we name all functions `getDesktopPath` (i.e: `getXXXPath`)? Sounds more consistent to me with the electron `getPath` naming. "Folder" is mostly a metaphor for end users, technically prefer using path/filename.

You can write `return electron.getPath('desktop') || '';` (i.e: `a || b` instead of `a ? a : b`).